.changeset/dry-drinks-love.md

```diff
@@ -0,0 +1,44 @@
+---
+"@salt-ds/core": minor
+---
+
+Add `Dialog`, `DialogHeader`, `DialogContent`, `DialogActions`, and `DialogCloseButton` to core
+Add patch to update dialog imports and `DialogHeader` renaming in mosaic-content-editor package
```

Comment:
This is not needed for consumer to know (i.e. when they're going to read the change log later).
Instead, this could be a good PR description